### ISSUE TITLE
Fine-tuning for novel view synthesis

### ISSUE THREAD

**Issue by zhongyingji** [facebookresearch/vggt#6]
Hi, thank you for your amazing work, it is really cool and inspiring. 

During the fine-tuning for novel view synthesis, the model is supervised by rendering loss on novel views. May I ask is the model only supervised by the rendering loss, or also supervised with additional camera/depth/point map loss? 

Thank you. : )

**Comment by jytime** [comment]
Hi,

Thanks for your interest. During finetuning, it was only supervised by the rendering loss in LVSM (photometric and lpips) :)

**Comment by zhongyingji** [comment]
Thank you for your rely!

**Comment by resurgo97** [comment]
@jytime I wonder if you are planning to release the weights of fine-tuned version for novel view synthesis task. 

Thanks.

**Comment by jytime** [comment]
Hi currently we do not have this plan

**Comment by Nickisdope** [comment]
@jytime Hi, I am wondering during NVS finetuning what is your training config? Do you only train the NVS head and keep the other parts frozen? Or do you learn the head with a large learning rate like 2e-4 and finetune the pretrained params with a smaller learning rate like 2e-5?
